### PR TITLE
TextInputType.number option added

### PR DESCRIPTION
**Pull request by gozeloglu**
I added `TextInputType.number` option for search bar. It is related to https://github.com/ArcticZeroo/flutter-search-bar/issues/32 issue. But, I wasn't able to run and test it. Because I got the following error when I tried to run. 

```
Launching lib\main.dart on AOSP on IA Emulator in debug mode...
Running Gradle task 'assembleDebug'...
√ Built build\app\outputs\flutter-apk\app-debug.apk.
Installing build\app\outputs\flutter-apk\app.apk...
E/AndroidRuntime(24565): FATAL EXCEPTION: main
E/AndroidRuntime(24565): Process: com.example.example, PID: 24565
E/AndroidRuntime(24565): java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{com.example.example/com.example.example.MainActivity}: java.lang.ClassNotFoundException: Didn't find class "com.example.example.MainActivity" on path: DexPathList[[zip file "/data/app/com.example.example-hx1kBentbJK6hGxXINBPzw==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.example-hx1kBentbJK6hGxXINBPzw==/lib/x86, /data/app/com.example.example-hx1kBentbJK6hGxXINBPzw==/base.apk!/lib/x86, /system/lib]]
E/AndroidRuntime(24565): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2843)
E/AndroidRuntime(24565): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
E/AndroidRuntime(24565): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
E/AndroidRuntime(24565): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
E/AndroidRuntime(24565): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
E/AndroidRuntime(24565): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
E/AndroidRuntime(24565): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/AndroidRuntime(24565): 	at android.os.Looper.loop(Looper.java:193)
E/AndroidRuntime(24565): 	at android.app.ActivityThread.main(ActivityThread.java:6669)
E/AndroidRuntime(24565): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(24565): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
E/AndroidRuntime(24565): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
E/AndroidRuntime(24565): Caused by: java.lang.ClassNotFoundException: Didn't find class "com.example.example.MainActivity" on path: DexPathList[[zip file "/data/app/com.example.example-hx1kBentbJK6hGxXINBPzw==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.example-hx1kBentbJK6hGxXINBPzw==/lib/x86, /data/app/com.example.example-hx1kBentbJK6hGxXINBPzw==/base.apk!/lib/x86, /system/lib]]
E/AndroidRuntime(24565): 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
E/AndroidRuntime(24565): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
E/AndroidRuntime(24565): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
E/AndroidRuntime(24565): 	at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:69)
E/AndroidRuntime(24565): 	at androidx.core.app.CoreComponentFactory.instantiateActivity(CoreComponentFactory.java:41)
E/AndroidRuntime(24565): 	at android.app.Instrumentation.newActivity(Instrumentation.java:1215)
E/AndroidRuntime(24565): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2831)
E/AndroidRuntime(24565): 	... 11 more
```

### CODE REVIEW

**Comment by gozeloglu** [comment]
Hi @ArcticZeroo . Can you review my pull request?

File: lib/src/flutter_search_bar_base.dart [hunk 1]
```diff
@@ -46,8 +46,8 @@ class SearchBar {
   /// A callback which is invoked each time the text field's value changes
   final TextFieldChangeCallback onChanged;
 
-  /// Whether or not the search bar should show up number keyboard. Defaults to 'false'.
-  final bool numberKeyboard;
+  /// What the keyboard type on the search bar. Defaults to 'TextInputType.text'.
```
Comment (ArcticZeroo) [inline lib/src/flutter_search_bar_base.dart:49]: This sentence doesn't really make grammatical sense, can you change it to this text I edited from the flutter docs?

`The type of keyboard to use for editing the search bar text. Defaults to 'TextInputType.text'.`

**Review by ArcticZeroo — CHANGES_REQUESTED** [review]
Code looks fine, just would like a small change in the docstring

File: lib/src/flutter_search_bar_base.dart (hunk 1 above)
Comment (gozeloglu) [reply lib/src/flutter_search_bar_base.dart]: I fixed in 6397f08

**Review by ArcticZeroo — APPROVED** [review]

**Pull request merged**